terragrunt/modules/crates-io/compute-static/Cargo.lock

```diff
@@ -0,0 +1,394 @@
+# This file is automatically @generated by Cargo.
```

Comment:
Is this lockfile actually used? The dependencies should be included in the one at the root of the repo due to the workspace.

Reply:
It wasn't until I added the external data source. But since Terragrunt copies the Rust crate into a temporary build environment, we cannot provide a stable path to include the crate in the workspace.